runtime/server/downloads.go

```diff
@@ -76,17 +76,18 @@ func (s *Server) downloadHandler(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	w.Header().Set("X-Content-Type-Options", "nosniff")
+	if request.Format == runtimev1.ExportFormat_EXPORT_FORMAT_CSV {
+		w.Header().Set("Content-Type", "text/csv")
+	} else {
+		w.Header().Set("Content-Type", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+	}
```

Comment:
Can we make the case explicitly check for `XLSX` and return an error if not CSV/XLSX? (e.g. in case data is corrupted)